src/test/java/org/takes/http/MainRemoteTest.java

```diff
@@ -62,6 +62,27 @@ public void exec(final URI home) throws IOException {
     }
 
     /**
+     * MainRemote passes additional arguments.
+     * @throws Exception If some problem inside
+     */
+    @Test
+    public void passesArgumentsToApp() throws Exception {
+        final String[] addArgs = {"works well!"};
```

Comment:
@HDouss just `args` won't be enough? http://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html